### DISCUSSION TITLE
Job is not Executed with JobDataMaps using jobStoreTX for Sql Server

### DISCUSSION THREAD

**Discussion by anandsb84** [quartznet/quartznet#1774] [Q&A]
Hi Marko,

Job without JobData it is executing fine.
but while passing JobData, it is not executing Job and there is no issue shown in the logger also.

IJobDetail demoJob = JobBuilder.Create().UsingJobData("className", "C1").WithIdentity("DemoJob")).RequestRecovery(true).Build();

Following is my Quartz configuration.

{"quartz.serializer.type","binary"},
{"quartz.jobStore.driverDelegateType" ,"Quartz.Impl.AdoJobStore.SqlServerDelegate, Quartz" },
{"quartz.jobStore.dataSource","default"},
{"quartz.jobStore.useProperties","true" },
{quartz.jobStore.type","Quartz.Impl.AdoJobStore.JobStoreTX, Quartz"},
{"quartz.dataSource.default.provider","SqlServer"},
{"quartz.dataSource.default.connectionString","(sqlConnectionString)"},

Could you please let me know what can be the reason behind this?

**Comment by lahma** [comment]
Duplicate of https://github.com/quartznet/quartznet/discussions/1765 ?
